wallet/how-to/connect/detect-wallet/multiple-wallets.md

```diff
@@ -0,0 +1,248 @@
+---
+description: Detect MetaMask and other wallets via EIP-6963.
+sidebar_position: 2
+---
+
+# Detect multiple wallets
+
+[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963) introduces an alternative discovery mechanism
+for multiple wallets, shifting from relying solely on [`window.ethereum`](metamask.md) for wallet detection.
+
+If a user has multiple wallet browser extensions installed, your web dapp can support
+[wallet interoperability](../../../concepts/wallet-interoperabilty.md) by adding support for
+EIP-6963, which enables your dapp to detect and connect to multiple installed wallets.
+You can [use third-party libraries](#use-third-party-libraries) or directly
+[implement EIP-6963](#implement-eip-6963) in your dapp.
+
+## Use third-party libraries
+
+You can add support for EIP-6963 by using the following third-party libraries:
+
+- [MetaMask SDK](../set-up-sdk/javascript/index.md)
+
+  :::note
+  MetaMask SDK does not support connecting to non-MetaMask wallets via EIP-6963.
+  If you intend to support discovery of other wallets, we recommend using other methods of adding
+  EIP-6963 support such as Wagmi 2+.
+  :::
+
+- [Wagmi 2+](https://wagmi.sh)
+
+- [Web3Modal 3+](https://docs.walletconnect.com/web3modal/about)
+
+- [MIPD Store](https://github.com/wevm/mipd)
+
+## Implement EIP-6963
+
+You can directly implement support for EIP-6963 in your React dapp.
+The steps are as follows:
```

Comment:
Maybe this can all be distilled down to...

To directly implement support for EIP-6963 in your React dapp: